packages/thirdparty-widgets/src/Map/index.tsx

```diff
@@ -1,27 +1,27 @@
+import { AlphaMapEditor } from "@brizy/builder-ui";
 import { Brizy } from "@brizy/core";
+import { helperHTML } from "../utils/constants";
+import { OutputOptionStyle } from "../utils/cssStyle/types";
+import { HOVER, NORMAL } from "../utils/stateMode";
+import "./index.css";
+import { Props } from "./types";
 
-export const mapComponent = (props: any) => {
-  const { address, zoom = 13 } = props;
-  const URL = "https://www.google.com/maps/embed/v1/place";
-  const KEY = "AIzaSyCcywKcxXeMZiMwLDcLgyEnNglcLOyB_qw";
-  const iframeSrc = `${URL}?key=${KEY}&q=${address}&zoom=${zoom}`;
-
-  return (
-    <div className="mapThirdComponent" style={{ pointerEvents: "none" }}>
-      <iframe className="" src={iframeSrc} title="map" />
-    </div>
-  );
-};
+export const mapComponent = ({ address, zoom }: Props): JSX.Element => (
+  <div className="brz-map brz-map_styles">
+    <AlphaMapEditor address={address} zoom={zoom} />
+  </div>
```

Comment:
need to create new component in `builder-ui` as example: `AlphaMapEditor2` and use it here